packages/auth-server/stores/client.ts

```diff
@@ -27,10 +27,10 @@ export const contractsByChain: Record<SupportedChainId, ChainContracts> = {
     accountPaymaster: "0x0cE4c9b3eA74971210C114cc136eAE41148C8B5a",
```

Comment:
Remaining work is to deploy new contracts, update the whitelist on Sepolia, and update these addresses